types/instance.d.ts

```diff
@@ -1,6 +1,6 @@
-import * as http from 'http'
 import { FastifyError } from '@fastify/error'
 import { ConstraintStrategy, HTTPVersion } from 'find-my-way'
+import * as http from 'http'
```

Comment:
This was moved due to VSCode's auto-import, should I leave it as-is, or move it back to the top? 😅

Reply:
I am ok with this change